third_party/closure/goog/caja/string/html/htmlparser.js

```diff
@@ -209,7 +209,6 @@ goog.string.html.HtmlParser.Elements = {
 /**
  * Regular expression that matches &s.
  * @type {RegExp}
- * @private
```

Comment:
I think the linter will complain that anything ending in `_` has to be private. So we probably have to rename these as well.